### PR TITLE
MySqlDriver,PgSqlDriver: Added info about foreign key behaviour

### PR DESCRIPTION
**Pull request by norbe**
It's based on code that is used be drivers at [adminer](adminer.org)

### CODE REVIEW

File: src/Database/Drivers/PgSqlDriver.php [hunk 1]
```diff
@@ -256,7 +257,18 @@ public function getForeignKeys($table)
 				co.contype = 'f'
 				AND cl.oid = {$this->connection->quote($this->delimiteFQN($table))}::regclass
 				AND nf.nspname = ANY (pg_catalog.current_schemas(FALSE))
-		")->fetchAll();
+		";
+
+		$keys = [];
+		foreach ($this->connection->query($query) as $row) {
+			if (preg_match('~FOREIGN KEY\s*\((.+)\)\s*REFERENCES (.+)\((.+)\)(.*)$~iA', $row['definition'], $match)) {
```
Comment (milo) [inline src/Database/Drivers/PgSqlDriver.php]: This is not systematic for PostgreSQL. There are columns for this purpose `confupdtype` and `confdeltype` (http://www.postgresql.org/docs/8.4/static/catalog-pg-constraint.html).

Reflection is, IMHO, internal thing and FK actions are not needed in Nette Database.

**Comment by norbe** [comment]
@milo I've updated implementation, thanks for link.

I can move it to our codebase if you think it's useless for others. I agree that it is not very common to use it, but I found that this info is already at SqliteDriver so I decided to create PR.

**Comment by Unlink** [comment]
I agree with milo, it's nice to have, but it increase cache file size and almost nobody needs it. If it is in SqliteDriver it should be IMHO removed too.

**Comment by dg** [comment]
Thanks, but this is really not needed by to be implemented by NDBT.

**Pull request closed without merging**